packages/web-react/src/hooks/useScrollControl.ts

```diff
@@ -17,15 +17,15 @@ const enableScroll = (offsetY: number) => {
   body.style.paddingRight = '';
   body.style.top = '';
   body.classList.remove(CLASSNAME_SCROLLING_DISABLED);
-  window.scrollTo(0, -offsetY);
+  window.scrollTo(0, offsetY);
```

Comment:
@crishpeen Let's discuss why there is here a negative value and what it caused. When used on Jobs.cz the Modal is rendered but not open. The `useEffect` hook is called causing the scroll to zero even if there is an `#` anchor in the URL and `window.offsetY` set to `5878`. This scroll must be saved and used.

Same if the Modal is open -> closed -> open: the page must stay on the same offset.

Reply:
I am not saying my code is flawless, probably the opposite, but with your updates our demo scrolls to the top on modal close. So I think it is still broken.

Reply:
I am not saying that this is the fix. This modification works on Jobs.cz side but I did not test it on our demo. Just saved the work to consult with you.

Reply:
Can you please explain the negative value for offset?

Reply:
Now when I look into it, it looks like it shouldn't be negative. 

You want to reset the position fixed (removing the class) and the body.top and then you want to scroll to the spot where you were before. So I think it shouldn't be negative.

Reply:
I have created and example with an anchor. Now I think I know why the value is negative.